<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>People_Email_Add</class>
 <widget class="QDialog" name="People_Email_Add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Email</string>
  </property>
  <widget class="QPushButton" name="AddEmail_CancelButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddEmail_FirstNameBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddEmail_SurnameBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddEmail_EmailBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="AddEmail_FirstNameText">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>First Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="AddEmail_SurnameText">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Surname</string>
   </property>
  </widget>
  <widget class="QLabel" name="AddEmail_EmailText">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Email Address</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddSong_SaveButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="Albums_MainMenuText">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add Email</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>AddEmail_CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>People_Email_Add</receiver>
   <slot>BackButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddSong_SaveButton</sender>
   <signal>clicked()</signal>
   <receiver>People_Email_Add</receiver>
   <slot>AddEmail()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>BackButton()</slot>
  <slot>AddEmail()</slot>
 </slots>
</ui>
